src/app/page/settings/settings.component.ts

```diff
@@ -36,6 +37,24 @@ export class SettingsComponent implements OnInit {
     });
   }
 
+  testnotify() {
+    return this.dataService.FireGET<INotification[]>(API.TestNotification).subscribe(result => {
+      for (const notification of result) {
+        if (notification.actor_type.includes('event')) {
+          notification.title = notification.actor.name;
+          notification.image_url = notification.actor.image_url || notification.actor.bodies[0].image_url;
+        } else if (notification.actor_type.includes('newsentry')) {
+          notification.title = notification.actor.title;
+          notification.image_url = notification.actor.body.image_url;
+        } else if (notification.actor_type.includes('blogentry')) {
+          notification.title = notification.actor.title;
+          notification.image_url = '/../../assets/logo.png';
+        }
+      }
+      this.dataService.notifications = result;
```

Comment:
Do we really need to update the notif list? The original purpose of the test notif was to test FCM.

@e-aakash